### ISSUE TITLE
What does Assembly v1.0 mean?

### ISSUE THREAD

**Issue by samanpwbb** [mapbox/assembly#789]
With https://github.com/mapbox/assembly/pull/787, everything in the v1.0 milestone will be complete. Are we actually ready for 1.0? 

To me, 1.0 means we're confident in the core concepts, and ready to make an announcement about this project to the world.

The one thing on the top of my mind that I'd seriously consider adding to 1.0 are the various icon issues (https://github.com/mapbox/assembly/issues/782, https://github.com/mapbox/assembly/issues/750, https://github.com/mapbox/assembly/issues/563).

I'm sure there are other things other folks have been thinking about as well, like font scale adjustments (cc @mgosline), or tweaks to the grid or flexbox handling.

I can make time for some of this work next week myself (unless someone like... @natslaughter wants to pitch in 😉 ). I don't want icons to hold us back from any internal Assembly use-cases, so we shouldn't necessarily wait on 1.0 to port Studio to Assembly.

cc: @mapbox/design @mapbox/studio

**Comment by davidtheclark** [comment]
Do we think that buttons are at a place where we'll actually use them as-is, instead of modifying them? It seems to me kind of counterproductive that we have a `btn` class that we never want to use as-is. And there's no question that people are going to build projects that use this class *without* additional modifications — so does that mean we should adjust it?

> The one thing on the top of my mind that I'd seriously consider adding to 1.0 are the various icon issues

I do think that we should either take care of those icon issues or close them. Letting them fester in the background won't do anything.

**Maybe we should consider a beta release.** Then we can try out the changes by porting existing work to the new stuff.

**Comment by samanpwbb** [comment]
> Do we think that buttons are at a place where we'll actually use them as-is, instead of modifying them?

what did you have in mind? Unrounding them? Reducing the font size?

**Comment by davidtheclark** [comment]
> what did you have in mind?

Oh, I'm fine with it the way it is — but think was thinking it's kind of unfortunate that what we actually use is different. This, for example, seems to be our default button style on dotcom:

![screen shot 2017-06-04 at 12 03 41 pm](https://cloud.githubusercontent.com/assets/628431/26764575/03993f06-491e-11e7-924f-afceca3f4481.png)

Smaller font size, larger left-right padding. Do you think we should make that the default? 

I'm ok with either way, really ... just a thought.

**Comment by samanpwbb** [comment]
Oh, yeah. I don't have strong opinions about this. If someone wants to submit a PR that changes the default font size for buttons I'd be cool with it but I don't think it's critical – that kind of customization on mapbox.com seems mostly fine to me (although the fact that we use 13px text but don't have it as a font size in assembly is kinda frustrating...).

**Comment by davidtheclark** [comment]
@katydecorah brought up a good idea about usage and versioning after 1.0: we could make a URL endpoint that serves the latest semver compatible version of Assembly. So you get the bug fixes, but you don't get the breaking changes.

**Comment by samanpwbb** [comment]
By the end of the week next week we should have Assembly at a place where we're ready to blog about 1.0 publicly. @mapbox/studio @mapbox/design if there are any final, breaking changes, lets get ticketed and taken care of soon.

**Comment by samanpwbb** [comment]
https://www.mapbox.com/assembly/ now reflects a version of assembly that includes everything in the 1.0 milestone. After we kick the tires with 0.14.0, we should be ready to release 1.0.

**Comment by davidtheclark** [comment]
![thanks-pickle](https://user-images.githubusercontent.com/628431/27878862-88da1db6-6174-11e7-9b37-c0289597ebe5.gif)

**Comment by samanpwbb** [comment]
Started a new branch for 1.0.0, only 4 years later! We've kicked the tires, and there are some improvements we're going to make over the next few weeks: https://github.com/mapbox/assembly/tree/1.0.0

**Comment by samanpwbb** [comment]
Closing, we'll have assembly 1.0.0 when we merge https://github.com/mapbox/assembly/pull/973
